daphne/src/lib.rs

```diff
@@ -744,6 +760,16 @@ impl DapAggregateShare {
             _ => return Err(DapError::fatal("invalid aggregate share merge")),
         };
 
+        if self.report_count == 0 {
+            // No interval yet, just copy other's interval
+            self.min_time = other.min_time;
+            self.max_time = other.max_time;
+        } else if other.report_count > 0 {
```

Comment:
This would be a bug, as it is important NOT to run the merge minimization when other.report_count == 0.  I will do this instead, as it makes the do nothing even clearer...

if self.report_count == 0 {
            // No interval yet, just copy other's interval
            self.min_time = other.min_time;
            self.max_time = other.max_time;
} else if other.report_count > 0 {
            // Note that we don't merge if other.report_count == 0, as in that case the timestamps
            // are 0 too, and thus bad to merge!
            self.min_time = min(self.min_time, other.min_time);
            self.max_time = max(self.max_time, other.max_time);
} else {
            // Do nothing!
}